libraries/entities-renderer/src/RenderablePolyVoxEntityItem.cpp

```diff
@@ -723,8 +723,10 @@ ShapeType RenderablePolyVoxEntityItem::getShapeType() const {
 }
 
 void RenderablePolyVoxEntityItem::setRegistrationPoint(const glm::vec3& value) {
-    if (value != _registrationPoint) {
```

Comment:
there's some cleanup below to make sure this stuff is all threadsafe and takes into account the pivot